packages/utils/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
please add this rule https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-default-export.md

we are going to use named export only as we agreed and new module is great start